templates/typescript/samples/sample-skill/src/bots/defaultActivityHandler.ts

```diff
@@ -61,36 +67,17 @@ export class DefaultActivityHandler<T extends Dialog> extends ActivityHandler {
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     protected async membersAdded(turnContext: TurnContext, next: () => Promise<void>): Promise<any> {
-        
-        const dc: DialogContext = await this.dialogs.createContext(turnContext);
-
-        if (dc.activeDialog !== undefined) {
-            return await dc.continueDialog();
-        } else {
-            return await dc.beginDialog(this.rootDialogId);
-        }
+        return DialogEx.run(this.dialog, turnContext, this.dialogStateAccessor);
```

Comment:
```suggestion
        await turnContext.sendActivity(this.templateEngine.generateActivityForLocale('IntroMessage');
        await DialogEx.run(this.dialog, turnContext, this.dialogStateAccessor);
```

Checking [this](https://github.com/microsoft/botframework-solutions/blob/v0.8-beta/samples/csharp/skill/SkillSample/Bots/DefaultActivityHandler.cs#L45)